### ISSUE TITLE
Adding a Wrapper to specific logger

### ISSUE THREAD

**Issue by rcalabro** [logtown/logtown#68]
Hi! I am trying this lib and came across this issue:

This logger works fine:
```
const Logger = require('logtown')

const getLogger = namespace => Logger.getLogger(namespace, {
  wrappers: [
    {
      log: (id, level, stats, ...rest) => {
        console.log(`${level} [${id}]`, ...rest)
      },
    },
  ],
})

module.exports = getLogger
```

But if I try to use the node wrapper included in the lib this the logger does not log anything:
```
const Logger = require('logtown')
const NodeWrapper = require('logtown/wrappers/node')

const nodeWrapper = new NodeWrapper()

const getLogger = namespace => Logger.getLogger(namespace, {
  wrappers: [nodeWrapper],
})

module.exports = getLogger
```

Although the node wrapper works if I add id globally 
`Logger.addWrapper(nodeWrapper)`
My use case requires different wrappers for each logger.

Any ideas?

**Comment by zemd** [comment]
@rcalabro 

just took a quick look and this bug happed after migration to deepmerge. will be fixed shortly. thanks for notifying.

**Comment by zemd** [comment]
also just want to notice that default wrappers are not so good for everyday usage. probably they require refactoring.

**Comment by zemd** [comment]
@rcalabro fixed in v`2.4.5`. please update and try again. closing for now.
